tests/migrations/test_operations.py

```diff
@@ -1334,6 +1339,60 @@ def test_alter_unique_together_remove(self):
         operation = migrations.AlterUniqueTogether("Pony", None)
         self.assertEqual(operation.describe(), "Alter unique_together for Pony (0 constraint(s))")
 
+    def test_create_index(self):
+        """
+        Tests the CreateIndex operation.
+        """
+        project_state = self.set_up_test_model("test_crin")
+        Pony = project_state.apps.get_model("test_crin", "Pony")
+        index = models.Index("pink", model=Pony)
```

Comment:
You shouldn't need to set the model here? You wouldn't have access to it in the migration file anyway. I think `state_forwards` should set up the connection for you